src/Tribe/Views/V2/Template/Title.php

```diff
@@ -108,6 +109,17 @@ public function build_title( $current_title = '', $depth = true ) {
 			);
 		}
 
+		/**
+		 * Filter the plural Events label for Views Title.
+		 *
+		 * @since TBD
+		 *
+		 * @param string  $events_label_plural The plural events label as it's been generated thus far.
+		 * @param Context $context             The context used to build the title, it could be the global one, or one externally
+		 *                                     set.
+		 */
+		$this->events_label_plural = apply_filters( 'tribe_events_filter_views_v2_plural_events_label', $this->events_label_plural, $context );
```

Comment:
should this happen before "Featured" is prefixed to the event label plural?

Reply:
Same as PRO, I moved so it works through the Filter.